mariadb/content.md

```diff
@@ -72,6 +72,20 @@ Note that users on host systems with SELinux enabled may see issues with this. T
 $ chcon -Rt svirt_sandbox_file_t /my/custom
 ```
 
+### Configuration without a `cnf` file
+
+Many configuration options can be passed as flags to `mysqld`. This will give you the flexibility to customize the container without needing a `cnf` file. For example, if you want to change the default encoding and collation for all tables to use `utf8` just run the following:
```

Comment:
IMO we should change "`utf8`" to "UTF-8 (`utf8mb4`)" and update the example below so that we're recommending "real" UTF-8, not the broken 3-byte version :smile: